quipucords/scanner/network/processing/test_installed_products.py

```diff
@@ -51,3 +47,22 @@ def test_failure_no_relevant_info():
         )
         == expected
     )
+
+
+def test_success_multiple_products():
+    """ID and Name match expected format for multiple products."""
+    stdout_output = (
+        "\r\nProduct:\r\n\tID: 479\r\n\tName: Red Hat Enterprise Linux for x86_64"
+        "\r\n\tVersion: 8.6\r\n--\r\n\r\nProduct:\r\n\tID: 69\r\n\tName: Red Hat"
+        " Enterprise Linux Server\r\n\tVersion: 8.7\r\n"
+    )
```

Comment:
not a blocker, but I'd rather read this as a multiline string. will be easier to read/maintain this.

Reply:
Same here 😍 would be easier to see the tested data.

Reply:
Your wish is my command, gents!